<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortTableWorkspaceDialog</class>
 <widget class="QWidget" name="SortTableWorkspaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sort Table Workspace</string>
  </property>
  <layout class="QVBoxLayout" name="dialogLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Input Workspace</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="MantidQt::MantidWidgets::WorkspaceSelector" name="workspace"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Output Workspace</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="output"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddColumn">
       <property name="toolTip">
        <string>Add a new column to sort by</string>
       </property>
       <property name="text">
        <string>Add column</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveColumn">
       <property name="toolTip">
        <string>Remove the last column as sorting key</string>
       </property>
       <property name="text">
        <string>Remove column</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Sort by columns</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>375</width>
           <height>196</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="columnsLayout" columnstretch="0,1,0">
            <item row="0" column="0">
             <widget class="QLabel" name="lblColumnName">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Column</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="cbColumnName">
              <property name="toolTip">
               <string>Sort the table by values in this column</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QComboBox" name="cbAscending">
              <property name="toolTip">
               <string>Sort order</string>
              </property>
              <item>
               <property name="text">
                <string>Ascending</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Descending</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::WorkspaceSelector</class>
   <extends>QComboBox</extends>
   <header>MantidQtMantidWidgets/WorkspaceSelector.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>workspace</tabstop>
  <tabstop>output</tabstop>
  <tabstop>btnAddColumn</tabstop>
  <tabstop>btnRemoveColumn</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>cbColumnName</tabstop>
  <tabstop>cbAscending</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
